src/wms/src/main/java/org/geoserver/wms/GetFeatureInfo.java

```diff
@@ -58,8 +58,14 @@ private FeatureCollectionType buildResults(List<FeatureCollection> results) {
 
     @SuppressWarnings("rawtypes")
     private List<FeatureCollection> execute(GetFeatureInfoRequest request) throws Exception {
+
         final List<MapLayerInfo> requestedLayers = request.getQueryLayers();
         FeatureInfoRequestParameters requestParams = new FeatureInfoRequestParameters(request);
+        // if url has a clip param and the clicked point is outside
```

Comment:
No commented out code please... believe this class needs no change at all now, right?

Reply:
will remove

Reply:
yes will replace with master version